css/style.css

```diff
@@ -89,6 +90,11 @@ a.logo:hover {
 
 h1.card-title {
   color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  order: 0;
+  flex-grow: 0;
 }
```

Comment:
- [ ] Good job on adding the margins, as the previous code reviewer mentions, please also add the border for the cards. 
- [Optional] Also consider adding [box-shadow](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow) for the cards to make them pop out even better.
### Your cards.
![Screenshot from 2022-09-23 14-31-19](https://user-images.githubusercontent.com/69077061/191951522-b86a4280-b9aa-4b1a-b6eb-a0412e640f17.png)
[Figma Cards](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

![Screenshot from 2022-09-23 14-31-42](https://user-images.githubusercontent.com/69077061/191951369-8dc9b653-3a3f-4c70-ab88-0324d289b6c0.png)